.pre-commit-config.yaml

```
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: "v3.1.0"
+    rev: "v4.0.0-alpha.8"
```

```suggestion
    rev: "v3.1.0"
```
please keep the original version. We want to use the currently stable version of prettier.

---

Hi Matthias, I am getting a lot of issues with unstable checksums although the same tests when run on my local system run smoothly. Not sure how to fix this

---

please revert this